script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -136,20 +136,20 @@ Golden hair and red eyes.
 -- Page 201, Offset 4484.
 -- @g　白い服。白い影。
 @g
-White clothes. A white figure.
+White clothes and white silhouette.
```

Comment:
Definitely needs an 'a' before white silhouette

Reply:
Problem with "white features" for me is that it actually does kinda sound a bit racially coded. I do agree that given silhouette is used later, maybe changing her is good. But figure doesn't quite do it for me.

Reply:
Pale features less coded?
> White clothes. Pale features.